packages/create-cloudflare/e2e-tests/frameworks.test.ts

```diff
@@ -123,7 +123,7 @@ describe.concurrent(`E2E: Web frameworks`, () => {
 			],
 			testCommitMessage: true,
 			timeout: LONG_TIMEOUT,
-			quarantine: true,
+			unsupportedOSs: ["win32"],
```

Comment:
This is something we'll need to investigate, it looks like `pnpm` on windows fails to build the solid app properly 😓

(It does feel like an upstream issue to me btw)

Reply:
Is it worth raising an issue on the SolidStart repo?